src/main/java/org/rocstreaming/roctoolkit/RocContext.java

```diff
@@ -34,7 +39,10 @@ public class RocContext extends NativeObject {
      */
     private static long tryOpen(RocContextConfig config) throws IllegalArgumentException, Exception {
         Check.notNull(config, "config");
-        return open(config);
+        LOGGER.log(Level.FINE, "roc_context opening");
```

Comment:
If it's easy to log configs when opening context/sender/receiver, let's do it. If it's not so easy, let's create an separate issue for that.

Reply:
done